resources/party.ts

```diff
@@ -181,12 +280,16 @@ export default class PartyTracker {
         nick: nick,
       };
     } else {
-      const jobName = Util.jobEnumToJob(partyMember.job);
-      const role = Util.jobToRole(jobName);
+      const job = Util.jobEnumToJob(partyMember.job);
+      const jobAbbr = jobLocalizedAbbr[job]?.[Options.AlertsLanguage ?? 'en'] ?? job;
+      const jobFull = jobLocalizedFull[job]?.[Options.AlertsLanguage ?? 'en'] ?? job;
+      const role = Util.jobToRole(job);
+      const roleName = roleLocalized[role]?.[Options.AlertsLanguage ?? 'en'] ?? role;
```

Comment:
```suggestion
      const lang = this.options.DisplayLanguage;
      const jobAbbr = jobLocalizedAbbr[job]?.[lang] ?? job;
      const jobFull = jobLocalizedFull[job]?.[lang] ?? job;
      const role = Util.jobToRole(job);
      const roleName = roleLocalized[role]?.[lang] ?? role;
```

The `Options` object is the default and not (necessarily?) the one with user config applied. It's better to use the one passed into the constructor here. I think DisplayLanguage is more correct here as `this.options` doesn't know about `AlertsLanguage` for oopsy. You'd need to add it to the `PartyTrackerOptions` type.

Reply:
(sorry for the complication, I forgot this part)